sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -35,7 +35,7 @@ object DottyPlugin extends AutoPlugin {
 
         // get latest nightly version from maven
         def fetchSource(version: String): (scala.io.BufferedSource, String) =
-          try Source.fromURL(s"https://repo1.maven.org/maven2/ch/epfl/lamp/dotty_$version/maven-metadata.xml") -> version
+          try Source.fromURL(s"https://repo1.maven.org/maven2/org/scala-lang/scala3_$version/maven-metadata.xml") -> version
```

Comment:
We probably shouldn't publish a scala3 artifact (the dotty artifact hasn't been useful in a lon gtime), so I would use `scala3-compiler_$version` here:
```suggestion
          try Source.fromURL(s"https://repo1.maven.org/maven2/org/scala-lang/scala3-compiler_$version/maven-metadata.xml") -> version
```